sdk/js/src/index.ts

```diff
@@ -467,7 +467,7 @@ export class Client {
     });
     if (response.error) {
       throw ClientError.newHttpError(
-        response.error.error,
```

Comment:
there are still a few other places where the `ClientError.newHttpError(response.error.error,...)` pattern is observed. any particular reason you didn't choose to modify those?

Reply:
e.g. in `submitOpportunity`, `getBids`, etc.

Reply:
fixed